streaming/src/main/java/io/gearpump/streaming/javaapi/Processor.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gearpump.streaming.javaapi;
+
+import io.gearpump.cluster.UserConfig;
+
+public class Processor<T extends Task> implements io.gearpump.streaming.Processor<T> {
+  private Class<T> _taskClass;
```

Comment:
LGTM. +1

The only one thing could be the leading underscore of all private properties. Is it a common java name convention, isn't it?